extensions/amp-ad-network-doubleclick-impl/0.1/amp-ad-network-doubleclick-impl.js

```diff
@@ -86,15 +89,12 @@ export class AmpAdNetworkDoubleclickImpl extends AmpA4A {
     // validateData, from 3p/3p/js, after noving it someplace common.
     const startTime = Date.now();
     const global = this.win;
-    const tagWidth = this.element.getAttribute('width');
-    const tagHeight = this.element.getAttribute('height');
-    let size;
-    if (tagWidth && tagHeight) {
-      size = `${tagWidth}x${tagHeight}`;
-    } else {
-      const slotRect = this.getIntersectionElementLayoutBox();
-      size = `${slotRect.width}x${slotRect.height}`;
-    }
+    const width = Number(this.element.getAttribute('width'));
```

Comment:
Sure, I can add that, but just FYI: the use of the slot attributes here predates this PR.